### PR TITLE
Feature/update cask confirmable module

### PR DESCRIPTION
**Pull request by ajainarayanan**

### CODE REVIEW

**Comment by ajainarayanan** [comment]
This PR is dependent on this [ng-capsules PR #11](https://github.com/caskdata/ng-capsules/pull/11)

**Comment by ajainarayanan** [comment]
This test is failing because it depends on this change https://github.com/caskdata/ng-capsules/pull/13/files.

File: coopr-ngui/app/main.js [hunk 1]
```diff
@@ -30,7 +30,8 @@ angular
 
       angular.module(PKG.name+'.filters', [
         PKG.name+'.services',
-        'cask-angular-capitalize'
+        'cask-angular-capitalize',
+        'cask-angular-confirmable'
```
Comment (gaarf) [inline coopr-ngui/app/main.js]: confirmable is not a filter, so IMHO it should go directly under .commons

Comment (ajainarayanan) [inline coopr-ngui/app/main.js]: :+1: Updated.

File: coopr-ngui/app/main.js [hunk 2]
```diff
@@ -44,7 +44,8 @@ angular
 
     ]).name,
 
-    'angular-loading-bar'
+    'angular-loading-bar',
+    'cask-angular-confirmable'
```
Comment (gaarf) [inline coopr-ngui/app/main.js]: now it's under the app... should go on line 35 ;-)

**Comment by rns4731** [comment]
LGTM if all tests pass

File: coopr-ngui/app/main.js (hunk 2 above)
Comment (ajainarayanan) [inline coopr-ngui/app/main.js]: Done. Fixed this.

**Comment by gaarf** [comment]
:+1:

File: coopr-ngui/package.json [hunk 3]
```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "start": "node ./server.js",
-    "build": "npm install && ./node_modules/bower/bin/bower install && node ./node_modules/gulp/bin/gulp.js build",
+    "build": "npm install && ./node_modules/bower/bin/bower install && ./node_modules/bower/bin/bower update && node ./node_modules/gulp/bin/gulp.js build",
```
Comment (gaarf) [inline coopr-ngui/package.json]: I think we can do just `update` as it includes `install`

Comment (ajainarayanan) [inline coopr-ngui/package.json]: :+1: Done. Fixed this.

**Comment by gaarf** [comment]
good to go pending travis :+1:

**Pull request merged**